style.css

```diff
@@ -0,0 +1,105 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  display: flex;
+  min-height: 100vh;
+  background: #666;
+  justify-content: center;
+  align-items: center;
+  font-family: "Lato", sans-serif;
+  font-weight: 400;
+ }
+
+.calculator {
+  position: relative;
```

Comment:
Why `position: relative;` here?

Reply:
to catch the child element with position absolute

Reply:
But you're not using `absolute` anymore